test/extensions/filters/common/ext_authz/ext_authz_http_impl_test.cc

```diff
@@ -546,8 +548,12 @@ TEST_F(ExtAuthzHttpClientTest, AuthorizationRequestError) {
   client_->onFailure(async_request_, Http::AsyncClient::FailureReason::Reset);
 }
 
-// Test the client when a call to authorization server returns a 5xx error status.
+// Test the client when a call to authorization server returns a 5xx error status and runtime guard is set to true
```

Comment:
This should be "false"?

Reply:
sure. will update this mas. thanks for noticing